modules/ROOT/pages/install-helm.adoc

```diff
@@ -157,7 +157,7 @@ metadata:
 subjects:
   - kind: ServiceAccount
     name: rtf-agent
-    namespace: rtf
+    namespace: <rtf-namespace>
```

Comment:
Again, this is inconsistent. In rtfctl, we are using `rtfNamespace` and here we are using `rtf-namespace`. Let's pick one and be consistent.